x-pack/plugins/ml/server/models/data_recognizer/modules/sample_data_weblogs/ml/low_request_rate.json

```diff
@@ -1,11 +1,12 @@
 {
-  "groups": ["kibana_sample_logs", "kibana_sample_data"],
-  "description": "Kibana sample web logs data: find anomalies in the visitor count",
+  "groups": ["kibana_sample_logs", "kibana_sample_web_logs"],
```

Comment:
We were planning on putting all the jobs for all the sample modules into the group `kibana_sample_data` as well as their own weblogs specific group. So I think this should be 
```
"groups": ["kibana_sample_web_logs", "kibana_sample_data"],
```  
(if you prefer `kibana_sample_web_logs` to `kibana_sample_logs` that was there before).